arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/EditFeaturesWithFeatureLinkedAnnotationViewController.swift

```diff
@@ -0,0 +1,228 @@
+        alert.textFields?[1].text = selectedFeature.attributes["ST_STR_NAM"] as? String
+        // Prompt the appropriate keyboard types.
+        alert.textFields?[0].keyboardType = .numberPad
+        alert.textFields?[1].keyboardType = .default
+        // Add a "Cancel" option and clear the selection if cancel is selected.
+        alert.addAction(UIAlertAction(title: "Cancel", style: .cancel) { _ in
+            self.clearSelection()
+            self.selectedFeature = nil
+        })
+        // Add a "Done" option to complete the editing process and close the alert.
+        alert.addAction(UIAlertAction(title: "Done", style: .default) { _ in
+            let addressTextField = alert.textFields?[0]
+            let streetTextField = alert.textFields?[1]
+            // Set the new values for the attributes and update the selected feature.
+            selectedFeature.attributes["AD_ADDRESS"] = Int((addressTextField?.text)!)
+            selectedFeature.attributes["ST_STR_NAM"] = streetTextField?.text
+            selectedFeature.featureTable?.update(selectedFeature)
+        })
+        // Present the alert.
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    // Move the currently selected point feature to the given map point, by updating the selected feature's geometry and feature table.
+    func movePoint(mapPoint: AGSPoint) {
+        guard let selectedFeature = self.selectedFeature else { return }
+        // Create an alert to confirm that the user wants to update the geometry.
+        let alert = UIAlertController(title: "Confirm update", message: "Would you like to update the geometry?", preferredStyle: .alert)
+        // Clear the selection and selected feature if "No" is selected.
+        alert.addAction(UIAlertAction(title: "No", style: .cancel) { _ in
+            self.clearSelection()
+            self.selectedFeature = nil
+        })
+        alert.addAction(UIAlertAction(title: "Yes", style: .default) { _ in
+            // Set the selected feature's geometry to the new map point.
+            selectedFeature.geometry = mapPoint
+            // Update the selected feature's feature table.
+            selectedFeature.featureTable?.update(selectedFeature) { (error: Error?) in
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    // Clear selection of polyline.
+                    self.clearSelection()
+                    self.selectedFeature = nil
+                }
+            }
+        })
+        // Present the alert.
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    // Move the last of the vertex point of the currently selected polyline to the given map point, by updating the selected feature's geometry and feature table.
+    func movePolylineVertex(mapPoint: AGSPoint) {
```

Comment:
The name of this function should be something like `moveLastVertexOfSelectedFeature(to:)`.